### PR TITLE
DEV: Add system test for updating color scheme of a theme

### PR DESCRIPTION
**Pull request by tgxworld**
Follow-up to 63119144ff481b1a18b69c0d638ece1b03c54766

### CODE REVIEW

File: spec/system/admin_customize_themes_spec.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      within(".theme-settings__color-scheme") do
+        expect(page).not_to have_css(".submit-edit")
+        expect(page).not_to have_css(".cancel-edit")
+
+        find(".color-palettes").click
+        find(".color-palettes-row[data-value='#{color_scheme.id}']").click
+        find(".submit-edit").click
+
+        expect(page).not_to have_css(".submit-edit")
+        expect(page).not_to have_css(".cancel-edit")
```
Comment (jjaffeux) [inline spec/system/admin_customize_themes_spec.rb:25]: This is slightly risky test which could stop testing what you think it is very easily.

If for some reason you don't end up on the page you think you should be after `find(".submit-edit").click` your test will still green.

Also Im not used to use `within()` but shouldn't we have these two expects outside of it?

File: spec/system/admin_customize_themes_spec.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    sign_in(admin)
+  end
+
+  describe 'when visiting the page to customize the theme' do
+    it 'should allow admin to update the color scheme of the theme' do
+      visit("/admin/customize/themes/#{theme.id}")
+
+      within(".theme-settings__color-scheme") do
+        expect(page).not_to have_css(".submit-edit")
+        expect(page).not_to have_css(".cancel-edit")
```
Comment (jjaffeux) [inline spec/system/admin_customize_themes_spec.rb:18]: Same question for this, is it supposed to be in the `within()` block?

Comment (jjaffeux) [reply spec/system/admin_customize_themes_spec.rb:18]: Im not exactly sure how `within()` works but I expect it to scope for things like `find` or `fillIn` but most likely not for `expect(page)`.

Comment (tgxworld) [reply spec/system/admin_customize_themes_spec.rb:18]: O you're right, `within` does nothing for page expectations or even page lookups

File: spec/system/admin_customize_themes_spec.rb (hunk 1 above)
Comment (tgxworld) [reply spec/system/admin_customize_themes_spec.rb]: Good point :+1:

**Comment by tgxworld** [comment]
@jjaffeux Can you have another look? I've rewritten the assertions to be scoped to the element of concern.

**Review by jjaffeux — APPROVED** [review]
looks great thx

**Comment by tgxworld** [comment]
Thank you for reviewing @jjaffeux

**Pull request merged**